File: C/fizz_buzz_neil_v.c
#include <stdio.h>

void solve_fizz_buzz(int n){
    
    if ( n%15 == 0 ){
        printf("Fizz Buzz");
        return;
    }
    
    if( n%3 == 0){
        printf("Fizz");
        return;
    }
    
    if( n%5 == 0){
        printf("Buzz");
        return;
    }
    
    printf("%d", n);
    
}

int main(){

    int n;
    scanf("%d", &n);
    
    for(int i = 1; i <= n; ++i){
        
        solve_fizz_buzz(i);
        
        if(i == n) printf(".\n");
        else printf(", ");
        
    }
    
    return 0;
    
}

File: Python/fizz_buzz_neil_v.py

def solveFizzBuzz(x):
    
    if(x % 15 == 0):
        print ("Fizz Buzz", end='')
        return

    if(x % 3 == 0):
        print ("Fizz", end='')
        return

    if(x % 5 == 0):
        print ("Buzz", end='')
        return

    print (x, end='')


n = int(input())

for i in range(1, n+1):
    solveFizzBuzz(i)
    if(i == n):
        print (".")
    else:
        print(",", end=' ')
